src/components/Footer/Footer.scss

```
@@ -0,0 +1,79 @@
+@import '../../styles/variable.scss';
+
+.footerWrapper {
+  position: fixed;
+  width: 100%;
+  bottom: 0;
+
+  .footerContainer {
+    @include display-flex(space-around, center);
+    flex-direction: row;
+    width: 100%;
+    height: 60px;
+    padding: 0 90px;
+    font-size: 15px;
+    border: 1px solid #ccc;
+
+    .snsIconSet {
+      flex: 1;
+      font-size: 20px;
+      color: rgb(147, 145, 145);
+
+      i {
+        padding-right: 10px;
+      }
+      i:last-child {
+        padding-right: 0;
+      }
```

```suggestion
      i {
        padding-right: 10px;
      
        &:last-child {
          padding-right: 0;
        }
```
& 연산자 사용하시면 좀 더 좋을거 같습니다 다음 PR에서 수정해주세요!